packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -1,9 +1,95 @@
-import React from 'react';
-import './my-profile.scss';
+import React        from 'react';
+import PropTypes    from 'prop-types';
 import { localize } from 'deriv-translations';
+import                   './my-profile.scss';
+
+// TODO: This is just mockup data. it will be removed after Adding BE API.
+const user_data = {
+    first_name                : 'johny',
+    last_name                 : 'bravo',
+    security_deposit          : 0,
+    buy_trades                : 23,
+    sell_trades               : 25,
+    past_month_trades         : 42,
+    past_month_completion_rate: 99,
+    average_release_time      : 5,
+    average_release_unit      : 'min',
+};
+
+function capitalize (string) {
+    if (!string) {
+        return '';
+    }
+    return string[0].toUpperCase() + string.slice(1);
+}
+
+const Cell = (props) => {
+    return (
+        <div className='my-profile__cell'>
+            <div className='my-profile__cell-data'>
+                <div className='my-profile__cell-data-main'>
+                    {props.value}
+                    <sub className="my-profile__cell-data-dimension" >
+                        {props.dimension}
+                    </sub>
+                </div>
+                <div className='my-profile__cell-data-sub'>
+                    <div className='my-profile__cell-data-sub-upper'>
+                        {props.upperSubValue}
```

Comment:
isn't this also a normal value? if yes then it should be snake case `_`

Reply:
It is a prop! isn't prop naming convention for React camelCase?! (e.g: `className`, `onChange` ...).

Reply:
only those that are exactly the same, like your examples. if it's a variable created by us, we follow snake case, if it's a function we use camelCase

Reply:
OK! I will change it. It is like writing PHP inside JS. :dizzy_face: